Gemfile

```
@@ -4,10 +4,10 @@ source 'https://rubygems.org'
 
 gemspec
 
-group :release do
+group :release, optional: true do
   gem 'faraday-retry', '~> 2.1', require: false
   gem 'github_changelog_generator', '~> 1.16.4', require: false
 end
 
 # Override gemspec for CI matrix builds.
-gem 'puppet', ENV.fetch('PUPPET_VERSION', '>= 7.24'), require: false
+gem 'openvox', ENV.fetch('PUPPET_VERSION', '>= 7.24'), require: false
```

@ekohl what's your opinion on this?
```suggestion
gem 'openvox', ENV['OPENVOX_VERSION'] || ENV.fetch('PUPPET_VERSION', '>= 7.24'), require: false
```